src/settings/configuration.rs

```diff
@@ -194,6 +201,11 @@ impl Configuration {
     #[must_use]
     pub fn combine(self, config: Configuration) -> Self {
         Self {
+            rule_selections: self
+                .rule_selections
+                .into_iter()
+                .chain(config.rule_selections.into_iter())
+                .collect(),
```

Comment:
I think this is backwards, since we want our selections to take precedence over those that we're merging in (and so, they have to come later).

Reply:
If I invert this, the Dagster repo (probably the most involved deployment) works as-is (no breaking change).

They have this config:

```toml
[tool.ruff]

# Extend root configuration.
extend = "../../pyproject.toml"

# Match black. Note that this also checks comment line length, but black does not format comments.
line-length = 88

# Use extend-ignore so that we ignore all the same codes ignored in root.
extend-ignore = [

  # (Unused import): When the same symbol is imported in multiple blocks, the
  # last import takes precedence for python. This causes Ruff to think an
  # import in an earlier block is unused and report F401.
  "F401",

  # (Redefinition): This happens frequently in docs_snippets when we import the same symbol in multiple
  # snippets within the same file.
  "F811",

  # (local variable assigned but never used): This happens a lot in docs snippets for didactic
  # purposes.
  "F841",

]
```

So, I guess that could now use `ignore` instead of `extend-ignore`, but it works either way.

Reply:
Ah yes ... very good catch! Thanks :) Fixed via force push.

Reply:
Nice that my assumption was correct in that it likely doesn't break real world configs :)